projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -624,7 +625,40 @@ function render_for_website( $data, $classes, $styles ) {
 			'success_message' => $data['success_message'],
 		)
 	);
-	?>
+
+	if ( Jetpack_Memberships::user_can_view_post() ) { ?>
+		<div <?php echo wp_kses_data( $data['wrapper_attributes'] ); ?>>
+		<div class="jetpack_subscription_widget">
+			<div class="wp-block-jetpack-subscriptions__container">
```

Comment:
Can everything up to here be outside the if so that we don't need to duplicate it? It's shared across both sides of the if/else statement.

Reply:
https://github.com/Automattic/jetpack/pull/33897/commits/a8be5d5d8bd15a6952ad83a6045cbc184eb90bbf